src/ui/utils/copyItems.ts

```diff
@@ -0,0 +1,58 @@
+import type {ConfigItem, ConfigItemData} from '@gravity-ui/dashkit';
+import {DashTabItemType} from 'shared/types';
+import type {ConnectionsData} from 'ui/units/dash/containers/Dialogs/DialogRelations/types';
+
+export const collectWidgetItemIds = (item: {
+    type: ConfigItem['type'];
+    data: ConfigItemData;
+    targetId?: string;
```

Comment:
Let's try to use `{targetId: string} | {id: string}` cause in other case we are suggesting that they are both optional